### PR TITLE
Deprecate device.FlashStore

### PR DESCRIPTION
**Pull request by kasperl**

### CODE REVIEW

File: system/extensions/esp32/wifi.toit [hunk 1]
```diff
@@ -278,12 +282,12 @@ class WifiModule implements NetworkModule:
     if dns_ips.size != 0:
       dns_module.dhcp_client_ = dns_module.DnsClient dns_ips
       if from_dhcp:
-        logger_.info "DNS server address dynamically assigned through dhcp" --tags={"dns": dns_ips}
+        logger_.info "dns server address dynamically assigned through dhcp" --tags={"ip": dns_ips}
```
Comment (floitsch) [inline system/extensions/esp32/wifi.toit:285]: Unrelated change.

**Review by floitsch — APPROVED** [review]
LGTM except for the unrelated change.

File: system/extensions/esp32/wifi.toit (hunk 1 above)
Comment (kasperl) [reply system/extensions/esp32/wifi.toit:285]: Yeah :)
